__tests__/lib/cli/format.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { requoteArgs } from 'lib/cli/format';
+
+describe( 'utils/cli/format', () => {
+	describe( 'requoteArgs', () => {
+		it.each( [
+			{
+				input: [ 'text with spaces' ],
+				expected: [ '"text with spaces"' ],
+			},
+			{
+				input: [ 'textnospaces' ],
+				expected: [ 'textnospaces' ],
+			},
+			{
```

Comment:
Just for the sake of being. thorough we may want to do other spaces variations, like `{ "json"  :  "string"    }`

Reply:
That's some good idea. I've added some extra space variations for JSON based on this comment.  🙇